### PR TITLE
WIP [Aftershock] Skullgun CBM

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary

SUMMARY: Mods "Aftershock: Skullgun CBM"

#### Purpose of change

You want cyberpunk you got cyberpunk!  What's more cyberpunk than a skullgun?  The Skullgun screams "I'm so cyberpunk slumgang that I clearly didn't think this through".

#### Describe the solution
Adds the skullgun CBM. For that last chance escape from the undead in your face!

#### Describe alternatives you've considered

Living a cyberpunk lie.

#### Testing

You want a gun in your head? You test it.  I just work in marketing.

### CODE REVIEW

File: data/mods/Aftershock/items/cbms.json [hunk 1]
```diff
@@ -54,5 +54,16 @@
     "price": 1000000,
     "price_postapoc": 100000,
     "difficulty": 8
-  }
+  },
+  {
+    {
```
Comment (anothersimulacrum) [inline data/mods/Aftershock/items/cbms.json:59]:
```suggestion
```

**Comment by Zireael07** [comment]
I am a simple man. I see "skullgun", I think "Gunther Hermann" xD

File: data/mods/Aftershock/items/gun/projectile.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "color": "magenta",
+    "skill": "shotgun",
+    "ammo": [ "shot" ],
+    "ranged_damage": { "damage_type": "bullet", "amount": -8 },
+    "dispersion": 855,
+    "durability": 6,
+    "clip_size": 1,
+    "reload": 200,
+    "flags": [ "NEVER_JAMS", "RELOAD_EJECT", "NO_UNWIELD", "TRADER_AVOID" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "shot": 1 } } ]
```
Comment (LaVeyanFiend) [inline data/mods/Aftershock/items/gun/projectile.json:23]:
```suggestion
    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "40": 1 } } ]
```

**Comment by Shodan14** [comment]
Can we get cranial bomb next?

**Comment by Maleclypse** [comment]
@shodan14 cranial bomb done.

**Comment by Shodan14** [comment]
> 
> 
> @Shodan14 cranial bomb done.

Amazing.

**Comment by I-am-Erk** [comment]
We're gonna need to do some strange backwards cross-talk on some of the weirder cbms once exodii are a thing.

**Comment by Maleclypse** [comment]
Closing now that the C++ PR has been merged.

**Pull request closed without merging**